prow/gerrit/adapter/trigger.go

```diff
@@ -48,7 +48,7 @@ func messageFilter(lastUpdate time.Time, change client.ChangeInfo, presubmits []
 			}
 			for _, presubmit := range presubmits {
 				j, ok := jobs[presubmit.Name]
-				if ok && strings.ToLower(j.State) == string(v1.FailureState) {
+				if ok && strings.ToLower(j.State) != string(v1.SuccessState) {
```

Comment:
isn't this will also retest triggered jobs?!

Reply:
fixed